manual/manual/cmds/comp.etex

```diff
@@ -100,6 +100,17 @@ phase can be run directly, as in:
 The produced file has the executable bit set, and it manages to launch
 the bytecode interpreter by itself.
 
+Upon successful completion of the type checking procedure, type information is
```

Comment:
There is a lack of introduction here: the previous paragraph was detailing the output of `ocamlc`, then this new paragraph is suddenly speaking of `type checking`. You should link the two by stating for instance that `ocamlc` is able to emit some information on its internal stages.